src/domains/ecommerce-app/api/ecommerce-webview/v2/_openapi.json.tpl

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
you should add terraform resources to create the v2 api version on api management too
the affected file is [this](https://github.com/pagopa/pagopa-infra/blob/main/src/domains/ecommerce-app/04_apim_ecommerce_webview.tf) (see how it have been done for v1 api version

Reply:
ok, i gave it a shot. let me know if its correct.
done here d8c3fe064599d6147f90d2d422ee8c6bc1ea9db9